ui/js/actions/file_info.js

```diff
@@ -112,12 +114,16 @@ export function doDeleteFile(outpoint, deleteFromComputer, abandonClaim) {
       }
     }
 
-    dispatch({
-      type: types.FILE_DELETE,
-      data: {
-        outpoint,
-      },
-    });
+    setTimeout(() => {
```

Comment:
This seems like a hack. Can the update to the progress instead be based on an event?